extensions/stackdriver/stackdriver_plugin_factory.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2019 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "extensions/common/wasm/null/null.h"
+#include "stackdriver.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace Wasm {
+namespace Null {
+namespace Plugin {
+namespace Stackdriver {
+NullVmPluginRootRegistry* context_registry_{};
+}  // namespace Stackdriver
+
+const std::string kStackdriverPluginName = "envoy.wasm.null.stackdriver";
+
+/**
+ * Config registration for a Wasm filter plugin. @see
+ * NamedHttpFilterConfigFactory.
+ */
+class StackdriverPluginFactory : public NullVmPluginFactory {
+ public:
+  StackdriverPluginFactory() {}
+
+  const std::string name() const override { return kStackdriverPluginName; }
```

Comment:
Actually I was complaining the signature, not your implementation before the update.
Please either revert or switch to constexpr char kStackdriverPluginName[] = "envoy.wasm.null.stackdriver";

https://github.com/envoyproxy/envoy/blob/master/STYLE.md
> non-PoD static and global variables are forbidden